.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
           RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${new_fix_notes}"
 
-          # Combine existing content with new fix notes
-          updated_content="${existing_content}<br/><br/>${RELEASE_NOTES}"
+          # Remove '##' from RELEASE_NOTES
+          RELEASE_NOTES=$(echo -e "${RELEASE_NOTES}" | sed 's/## //g')
+
+          # Remove extra HTML
+          RELEASE_NOTES=$(echo -e "${RELEASE_NOTES}" | sed -r 's/<[^>]*>//g')
+
+          # Replace markdown links with confluence links
+          RELEASE_NOTES=$(echo -e "${RELEASE_NOTES}" | sed -r 's/\[([^]]+)\]\(([^)]+)\)/[\1|\2]/g')
+
```

Comment:
- For the final step where you combine the existing content with the new fix notes, you might want to consider using a more structured format (like JSON or YAML) for storing these notes. This would make the process of combining and formatting these notes more reliable and less error-prone. This is just a suggestion, and it might require significant changes to other parts of your codebase.